ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -7,41 +7,59 @@ import UIKit
 
 /**
  `CommandBar` is a horizontal scrollable list of icon buttons divided by groups.
- Provide `itemGroups` in `init` to set the buttons in the scrollable area. Optional `leadingItem` and `trailingItem` add fixed buttons in leading and trailing positions. Each `CommandBarItem` will be represented as a button.
  Set the `delegate` property to determine whether a button can be selected and deselected, and listen to selection changes.
+ Provide `itemGroups` in `init` to set the buttons in the scrollable area. Optional `leadingItemGroups` and `trailingItemGroups` add buttons in leading and trailing positions. Each `CommandBarItem` will be represented as a button.
  */
 @objc(MSFCommandBar)
 open class CommandBar: UIView {
     // Hierarchy:
     //
-    // leadingButton
-    // containerView
-    // |--layer.mask -> containerMaskLayer (fill containerView)
-    // |--subviews
-    // |  |--scrollView (fill containerView)
-    // |  |  |--subviews
-    // |  |  |  |--stackView
-    // |  |  |  |  |--buttons (fill scrollView content)
-    // trailingButton
+    // commandBarContainerStackView
+    // |--leadingCommandGroupsView
+    // |--|--buttons
+    // |--containerView
+    // |--|--layer.mask -> containerMaskLayer (fill containerView)
+    // |--|--subviews
+    // |--|  |--scrollView (fill containerView)
+    // |--|  |  |--subviews
+    // |--|  |  |  |--stackView
+    // |--|  |  |  |  |--buttons (fill scrollView content)
+    // |--trailingCommandGroupsView
+    // |--|--buttons
 
     // MARK: - Public methods
 
-    @objc public init(itemGroups: [CommandBarItemGroup], leadingItem: CommandBarItem? = nil, trailingItem: CommandBarItem? = nil) {
-        self.itemGroups = itemGroups
-
-        super.init(frame: .zero)
+    @available(*, renamed: "init(itemGroups:leadingItemGroups:trailingItemGroups:)")
+    @objc public convenience init(itemGroups: [CommandBarItemGroup], leadingItem: CommandBarItem? = nil, trailingItem: CommandBarItem? = nil) {
+        var leadingItems: [CommandBarItemGroup]?
+        var trailingItems: [CommandBarItemGroup]?
 
         if let leadingItem = leadingItem {
-            self.leadingButton = button(forItem: leadingItem, isPersistSelection: false)
+            leadingItems = [[leadingItem]]
         }
+
         if let trailingItem = trailingItem {
-            self.trailingButton = button(forItem: trailingItem, isPersistSelection: false)
+            trailingItems = [[trailingItem]]
         }
 
-        translatesAutoresizingMaskIntoConstraints = false
+        self.init(itemGroups: itemGroups, leadingItemGroups: leadingItems, trailingItemGroups: trailingItems)
+    }
+
+    @objc public init(itemGroups: [CommandBarItemGroup], leadingItemGroups: [CommandBarItemGroup]? = nil, trailingItemGroups: [CommandBarItemGroup]? = nil) {
```

Comment:
nit: can you break the parameters into multiple lines for better readability? (here and all the other func calls or definitions of the PR)

Reply:
Updated where it seemed necessary.